turbo.json

```diff
@@ -1,5 +1,6 @@
 {
     "$schema": "https://turbo.build/schema.json",
+    "globalEnv": ["CI"],
```

Comment:
This will disable remote caching on PRs, forever. Locally run builds will never match ones run on PRs.